documentation/plugins/index.markdown

```diff
@@ -11,3 +11,214 @@ title: Plugins
 
 MoveIt connects a number of different fields and provides a lot of interfaces so that users can override different parts of the framework and implement new concepts without having to touching the core of the framework.
 The following lists the different types of plugins and provides an overview over the use of them within the framework.
+
+* [move_group::MoveGroupCapability](#MoveGroupCapability)
+* [kinematics::KinematicsBase](#KinematicsBase)
+* [planning_interface::PlannerManager](#PlannerManager)
+* [moveit_controller_manager::MoveItControllerManager](#MoveItControllerManager)
+* [moveit_ros_control_interface::ControllerHandleAllocator](#ControllerHandleAllocator)
+* [moveit_sensor_manager::MoveItSensorManager](#MoveItSensorManager)
+* [collision_detection::CollisionPlugin](#CollisionPlugin)
+
+TODO:
+* [occupancy_map_monitor::OccupancyMapUpdater](#OccupancyMapUpdater)
+* [planning_request_adapter::PlanningRequestAdapter](#PlanningRequestAdapter)
+* [constraint_samplers::ConstraintSamplerAllocator](#ConstraintSamplerAllocator)
+
+Deprecated:
+
+* [planning_interface::Planner](#Planner)
```

Comment:
can you open an issue to remove this from the code base, pointing to where the interface is